apps/prairielearn/src/pages/instructorFileBrowser/instructorFileBrowser.html.ts

```diff
@@ -196,6 +198,7 @@ export function InstructorFileBrowser({
         ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", resLocals)}
         <main id="content" class="container-fluid">
           ${syncErrorsAndWarnings}
+          <h1 class="sr-only">File Browser</h1>
```

Comment:
Nit, probably use the same heading for both authorized and non-authorized views?

Reply:
Updated in [af07a83](https://github.com/PrairieLearn/PrairieLearn/pull/10440/commits/af07a83e2931531fae4210753b6f9e267a4b0567).

Reply:
I would in fact go the other way and standardize on "Files" (that is, keep the existing visual title, and make the screenreader-only title match it).

Reply:
Did you mean something like this, [38fde3a](https://github.com/PrairieLearn/PrairieLearn/pull/10440/commits/38fde3aac2c083893cff8c78824c85fda866dfec)?

Reply:
Exactly!